refinery/tool_manager/tests.py

```diff
@@ -1694,25 +1999,43 @@ def test_is_tool_based(self):
         self.create_valid_tool(ToolDefinition.WORKFLOW)
         self.assertTrue(self.tool.analysis.is_tool_based)
 
+    @mock.patch(
```

Comment:
In general, it would be great if I could make these mocks once in a setup or `ToolManagerTestBase`